main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
```

Comment:
@le717 You haven't asked dumb questions... No need to apologize!  Coding tired sucks and I keep doing it everyday... :)